browser/brave_wallet/keyring_controller_unittest.cc

```diff
@@ -1176,16 +1184,22 @@ TEST_F(KeyringControllerUnitTest, SetDefaultKeyringDerivedAccountName) {
   const std::string name2 = "Account2";
   const std::string account_path2 = KeyringController::GetAccountPathByIndex(1);
 
-  KeyringController::SetAccountNameForKeyring(GetPrefs(), account_path1, name1,
```

Comment:
test case should also be renamed to `SetDefaultKeyringDerivedAccountMeta`

Reply:
rebased in ac54b21e5b6da2e3faf2d4b13c268244326abde4